CHANGELOG-3.1.md

```diff
@@ -1,4 +1,16 @@
-## [v3.1.12](https://github.com/coreos/etcd/releases/tag/v3.1.12) (TBD)
+
+## [v3.1.12](https://github.com/coreos/etcd/releases/tag/v3.1.13) (TBD)
+
+See [code changes](https://github.com/coreos/etcd/compare/v3.1.12...v3.1.13) and [v3.1 upgrade guide](https://github.com/coreos/etcd/blob/master/Documentation/upgrades/upgrade_3_1.md) for any breaking changes.
+
+### Improved
+
+- Adjust [election timeout on server restart](https://github.com/coreos/etcd/pull/9364) to reduce [disruptive rejoining servers](https://github.com/coreos/etcd/issues/9333).
```

Comment:
@gyuho Happy to discuss if this should go all the way back to 3.1 or not.

Reply:
Yeah, I'm fine with backporting to 3.1.

Reply:
Original issue from @wojtek-t was running 3.1 https://groups.google.com/forum/#!msg/etcd-dev/82bPTmzGVM0/EiTINb6dBQAJ.